.travis.yml

```diff
@@ -20,7 +20,12 @@ env:
 
 before_install:
 - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo add-apt-repository -y ppa:george-edison55/precise-backports; fi
```

Comment:
Can some of these improvements not be made with whatever version travis has? We rely on cmake on a lot of platforms, and requiring people to upgrade what they have installed (especially if it is not under their control) is really not desirable. Would like to keep cmake requirements low unless there is a VERY compelling argument.